### PR TITLE
Add section for SELinux troubleshooting

### PR DESCRIPTION
**Pull request by nachtjasmin**
## Proposed change

I had a couple of problems when I tried to run paperless-ngx on my homeserver (running Fedora Server 38) and since it's running now, I just added a couple of steps that might help others. ^^

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other (please explain):

Added SELinux-specific troubleshooting stuff.

## Checklist:

- [x] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [ ] If applicable, I have included testing coverage for new code in this PR, for [backend](https://docs.paperless-ngx.com/development/#testing) and / or [front-end](https://docs.paperless-ngx.com/development/#testing-and-code-style) changes.
- [ ] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [ ] If applicable, I have checked that all tests pass, see [documentation](https://docs.paperless-ngx.com/development/#back-end-development).
- [x] I have run all `pre-commit` hooks, see [documentation](https://docs.paperless-ngx.com/development/#code-formatting-with-pre-commit-hooks).
- [x] I have made corresponding changes to the documentation as needed.
- [ ] I have checked my modifications for any breaking changes.

### CODE REVIEW

**Comment by shamoon** [comment]
I have no issue with this in general except that it’s basically gibberish to me and so I have no way of knowing how generalizable this is to others. Maintaining documentation for hyper specific things like this can be problematic because other users will have thoughts / want to make changes etc. and then we’re stuck having to administer information we can’t really verify

Alternative would be to add a link to a (presumably new) page in the wiki which allows users to edit it as they see fit. That’s what we’ve done for example with reverse proxy set up information. to be honest, I think that would be a more durable solution but not sure what others think

**Comment by stumpylog** [comment]
I'm also not sure how general this advice is to most users.  It really seems like just "Docker on SELinux" hiccups anyway, not paperless.

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -352,3 +352,54 @@ This is intentional as the output archive file may differ in unexpected or undes
...
+
+```shell
+$ sudo semanage fcontext --add --type container_file_t "/etc/paperless/consume(/.*)?"
+```
+
+After you added the SELinux override, it's time to relabel the directory and all subfolders and -files.
+
+```shell
+$ sudo restorecon -Rv /etc/paperless/consume
+```
```
Comment (shamoon) [inline docs/troubleshooting.md:405]: ## Platform-Specific Deployment Troubleshooting

A user-maintained wiki page is available to help troubleshoot issues that may arise when trying to deploy Paperless-ngx on specific platforms, for example SELinux. Please see [the wiki](https://github.com/paperless-ngx/paperless-ngx/wiki/Platform%E2%80%90Specific-Troubleshooting).

**Review by shamoon — APPROVED** [review]
@nachtjasmin Appreciate you taking the time to write this up so I've gone ahead with the suggestion and added a link to the wiki here with your information. Of course, you can now feel free to edit that wiki page.

Thanks again

**Pull request merged**